src/screen/Coupons.js

```diff
@@ -1,12 +1,97 @@
 import React, {Component} from 'react';
-import {Text} from 'react-native';
+import {FlatList} from 'react-native';
+import CouponRow from '../component/CouponRow';
+import CouponHeader from '../component/CouponHeader';
 
 export default class Coupon extends Component {
     static navigationOptions = {
         title: 'I tuoi buoni',
     };
 
     render() {
-        return <Text>Buoni</Text>;
+        let headerTextStart,
+            headerTextQuantity = '1 buono',
+            headerTextEnd = ' ancora da spendere';
+        if (headerTextQuantity == '1 buono') {
```

Comment:
You should only use the number of coupons, not comparing strings.

Get unused coupons with filter/length over `coupons` array.